modules/machine-delete.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * machine_management/delete-machine.adoc
+
+[id="machine-delete_{context}"]
+= Deleting a specific machine
+
+You can delete specifics machine.
+
+.Prerequisites
+
+* Install an {product-title} cluster and the `oc` command line.
+* Log into `oc` as a user with `cluster-admin` permission.
+
+.Procedure
+
+. View the Machines that are in the cluster:
++
+----
+$ oc get machine -n openshift-machine-api
+----
++
+The Machines are listed in the form of `<clusterid>-worker-<aws-region-az>`.
+
+. Delete the machine:
++
+----
+$ oc delete <machine> -n openshift-machine-api
+----
+
++
+[IMPORTANT]
+====
+By default, the machine controller will try to drain the node backed by the machine until if succeeds. There are situations where draining is never able to succeed, e.g a bad pod disruption budget configuration. You can opt-out from draining by setting "machine.openshift.io/exclude-node-draining" in a specific machine.
```

Comment:
```suggestion
By default, the machine controller tries to drain the node that is backed by the machine until it succeeds. In some situations, such as with a misconfigured pod disruption budget, where the drain operation cannot succeed. You can skip draining the node by setting "machine.openshift.io/exclude-node-draining" in a specific machine.
```

How do you know that you need to do this? Is there a downside to skip draining the node before you delete it? If not, this should be a step after you view the machine.